packages/es-components/src/components/containers/notification/Notification.md

```diff
@@ -1,239 +1,100 @@
-### Notification Types
+Notifications are static containers that will render any children passed to it.
+
+There are two additional components `InlineMessage` and `Message` that can be used to display text in a `Notification`.
+
+```jsx
+const Message = require('./Message').Message;
+const InlineMessage = require('./Message').InlineMessage;
 
-```
 <div>
   <Notification
     type="success"
-    header="Success!"
-    additionalText="You did a thing."
-  />
+  >
+    <Message emphasizedText="Success" text="This is a successful notification!" />
```

Comment:
Why wouldn't we just allow whatever content within the notification? I don't see a lot of use for the Message or InlineMessage components.

Reply:
We allow any content within the notification. Those two components are to have some built in messaging structure in a component that can be used. They don't have to be used at all.

Reply:
Gotcha... and now I see your explanation in the top post as well. Thanks!